src/lib/ProductExporter.ts

```diff
@@ -49,8 +49,16 @@ export class ProductExporter {
         this.productList = products;
     }
 
-    public async all(conn: core.Connection) {     
-        // Util.removeDir(this.dir);    <- ziom nie usuwaj katalogu
+    public async all(conn: core.Connection) {
+        if(this.productList[0] === '*ALL'){
+            this.productList = new Set<string>();
+            let productList = await Queries.queryAllProductNames(conn);
+            Util.isBulkApi(productList) ? productList = await Queries.bulkQueryAllProductNames(conn) : null;
```

Comment:
@adrianlukaszek1992 productName'ów na pewno nie trzeba querować bulk API, to proste query, wyciągasz kilka pól, nawet jeśli tych produktów będzie sporo to REST sobie poradzi